packages/web/src/components/notification/Notifications/MilestoneNotification.tsx

```diff
@@ -19,13 +24,54 @@ const messages = {
   reached: 'has reached over'
 }
 
+export const formatAchievementText = (
+  type: string,
+  name: string,
+  value: number,
+  achievement: string
+) => {
+  const achievementText =
+    achievement === Achievement.Listens ? 'Plays' : achievement
+  return `My ${type} ${name} has more than ${value} ${achievementText} on @AudiusProject #Audius
+Check it out!`
+}
+
+const getAchievementText = async (notification: Milestone) => {
+  const { achievement, user, value } = notification
+  switch (achievement) {
+    case Achievement.Followers: {
+      const link = fullProfilePage(user.handle)
+      const text = `I just hit over ${value} followers on @AudiusProject #Audius!`
```

Comment:
another one for messages probably. there are other ones too, basically a lot of the `text = ...` or `text: ...`
i'm sure you've seen them but calling out just in case